openshift-container-platform-4/policies/CM-Configuration_Management/component.yaml

```diff
@@ -978,22 +978,43 @@
   narrative:
```

Comment:
should we move the implementation_status to `complete` then? If we can already do this in OpenShift.